compiler/rustc_parse/src/parser/ty.rs

```diff
@@ -1112,8 +1120,15 @@ impl<'a> Parser<'a> {
         &mut self,
         lo: Span,
         parens: ast::Parens,
+        in_supertrait_context: bool,
         leading_token: &Token,
     ) -> PResult<'a, GenericBound> {
+        let auto_impl = if in_supertrait_context && self.eat_keyword(exp!(Auto)) {
+            self.expect_keyword(exp!(Impl))?;
```

Comment:
What is `auto impl Trait`?
I don't see it in tests.
Is it a part of the "supertrait items in subtrait impls" feature, or some other related feature?